content/en/glossary/terms/pipeline_execution_time.md

```diff
@@ -0,0 +1,9 @@
+---
+title: pipeline execution time
+core_product:
+  - ci-cd
+---
+Pipeline execution time is the amount of time a pipeline has been running jobs.
+For example, consider a pipeline with 3 jobs: A, B, and C. During the first two minutes both A and B are running, so those two minutes count towards execution time. The next 30 seconds, A stops while B is still running, so that time also counts. Then, there is a gap until C starts running, so the execution time is not increased during this period since there are no jobs running. Finally, C runs for one minute. The final execution time calculation is 2m30s + 1m = 3m30s.
```

Comment:
```suggestion
For example, consider a pipeline with 3 jobs: A, B, and C. During the first minute both A and B are running, so those one minute counts towards execution time. The next 30 seconds, A stops while B is still running, so that time also counts. Then, there is a gap of 5m until C starts running, so the execution time is not increased during this period since there are no jobs running. Finally, C runs for one minute. The final execution time calculation is 1m + 30s + 1m = 2m30s.
```

The "two minutes count" can easily be misinterpreted if you misread the beginning. Let's simplify it by only using one minute where A and B run

Reply:
I prefer to leave it as 2 minutes and then 1 minute since then it is more "asymmetric". Beceause if it is 1 minute then a gap and then 1 minute it might be harder to connect the dots IMO

Reply:
Here is the drawing I did:
![image](https://github.com/DataDog/documentation/assets/124577199/a4ed311d-707d-4e4c-aaa8-5fdc45fbecc5)

Reply:
> I prefer to leave it as 2 minutes and then 1 minute since then it is more "asymmetric". Beceause if it is 1 minute then a gap and then 1 minute it might be harder to connect the dots IMO

I understand. Let's reduce C to a 15s run so we can keep A+B on 1m

I am really concerned that reading fast people will understand "Ok A and B counts for 2 minutes so they sum parallelized work"

Reply:
Okey that works